node_package/src/clientStartup.js

```diff
@@ -63,6 +63,10 @@ function turbolinksVersion5() {
   return (typeof Turbolinks.controller !== 'undefined');
 }
 
+function turbolinksSupported() {
+  return Turbolinks.supported;
```

Comment:
> 1. Is this compatible with v2 and v5 turbolinks?

refer to turbolinks/turbolinks and turbolinks/turbolinks-classic repo and it's README.md,

- v2 is export `Turbolinks.supported` since https://github.com/turbolinks/turbolinks-classic/commit/46bc7bf4353aa8355d0950454687ab12be659e96.
- v5 also export it since first beta release (https://github.com/turbolinks/turbolinks/commit/0b9b0846e3b7543e692606beadbe833d5b793bc4). 👍

> 2. What browsers are not supported?

A browser that doesn't support`history.pushState` nor `requestAnimationFrame`, `Turbolinks.supported` will return `false`. e.g. IE9-IE10 😩 

In my use case, I don't find suitable polyfills/shims for that browser APIs. 
But [ReactDOM still supporting that old browsers](https://facebook.github.io/react/blog/2016/01/12/discontinuing-ie8-support.html), so React itself works fine.